sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -76,6 +78,23 @@ public void verbose(String message) {
         }
     }
 
+    /**
+     * Logs a message at {@code verbose} log level.
+     *
+     * <p><strong>Code samples</strong></p>
+     *
+     * <p>Logging a message at verbose log level.</p>
+     *
+     * {@codesnippet com.azure.core.util.logging.clientlogger.verbose}
```

Comment:
Right now this will be copying the codesnippet showing how to use the logger with the `String` overload. Let's create new codesnippets showing off the `Supplier` based overloads. You'll find the file containing these samples [here](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/core/azure-core/src/samples/java/com/azure/core/util/logging/ClientLoggerJavaDocCodeSnippets.java).

Let's create these new samples using the names `com.azure.core.util.logging.clientlogger.<API name>#Supplier`

Reply:
fixed